ccmlib/scylla_docker_cluster.py

```diff
@@ -44,7 +44,7 @@ def create_node(self, name, auto_bootstrap, thrift_interface,
                                 initial_token, save=save, binary_interface=binary_interface,
                                 scylla_manager=self._scylla_manager)
 
-    def _update_config(self):
+    def _update_config(self, install_dir=None):
```

Comment:
why are you adding those ? i.e. they are not in use

Reply:
This function overrides Cluster._update_config, that has this parameter.

Reply:
> why are you adding those ? i.e. they are not in use

@fruch @fgelcer 
If the upgrade test will run from the docker, I think you need to handle install_dir like it done in the Cluster._update_config

Reply:
that would be a problem, since we don't have install dir when using docker.
FYI upgrades using docker, can use real RPMs and install them into the docker

but we'll cross that bridge when we'll get to ti